src/models/mlp.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
+from torch.nn import Module, Linear, ReLU, Dropout, Sigmoid, Sequential, BCELoss
+import torch.optim as optim
+from skorch import NeuralNetClassifier
+
+class MLPModule(Module):
```

Comment:
Later on, nothing is critical no worries. Name it so that we know it's a Longitudinal variant. If it appears to be relevant to your MSc, Alex will undoubtedly provide you with a name; however, as of today, it is unknown. As a result, simply use LongitdinalMLP or whatever you deem more appropriate, but only to distinguish it's not pure MLP. Though nothing is critical.

Reply:
Understood, I'll just rename to TemporalMLP and can work from there (so it's clearly labelled)